app/css/animator.css

```diff
@@ -444,9 +440,7 @@ a {
   -webkit-appearance: none;
   margin-top: -9.5px;
 }
-#input-onion-skin-opacity:focus::-webkit-slider-runnable-track {
-  background-color: #D9D9D9;
-}
+#input-onion-skin-opacity:focus::-webkit-slider-runnable-track { background-color: #D9D9D9; }
```

Comment:
Just because this selector is so long, I would be fine with keeping it in the expanded format.